### PR TITLE
[v6r18] Test fixes

### PR DESCRIPTION
**Pull request by fstagni**

### CODE REVIEW

**Comment by fstagni** [comment]
Review anyone?

File: Interfaces/API/Dirac.py [hunk 1]
```diff
@@ -871,11 +871,13 @@ def runLocal(self, job):
           shutil.copytree( isFile, os.path.basename( isFile ), symlinks = True )
         elif os.path.exists( isFile ):
           shutil.copy( isFile, os.getcwd() )
+        elif os.path.exists(tmpdir + os.path.sep + isFile): # if it is in the tmp dir
```
Comment (chaen) [inline Interfaces/API/Dirac.py]: ```os.path.join``` ?

File: WorkloadManagementSystem/Service/MatcherHandler.py [hunk 2]
```diff
@@ -104,25 +102,27 @@ def export_requestJob( self, resourceDescription ):
 
 ##############################################################################
   types_getActiveTaskQueues = []
-  def export_getActiveTaskQueues( self ):
+  @staticmethod
+  def export_getActiveTaskQueues():
```
Comment (chaen) [inline WorkloadManagementSystem/Service/MatcherHandler.py]: are we sure that this works with DISET ?

Comment (fstagni) [inline WorkloadManagementSystem/Service/MatcherHandler.py]: Sure, there are several examples already

**Pull request merged**